src/main/java/teammates/ui/controller/InstructorFeedbackEditPageData.java

```diff
@@ -309,6 +309,15 @@ public void setStatusForAjax(String statusForAjax) {
         this.statusForAjax = statusForAjax;
     }
 
+    /**
+     * Retrieves the link to submit the request for copy of session. 
+     * Also contains feedback page link to return after the action.
+     * @return form submit action link
+     */
+    public String getEditCopyActionLink() {
+        return editCopyActionLink;
```

Comment:
and I saw a similar question on stackoverflow and it says the target container must be able to support invoking methods with arguments and can be done with all Servlet 3.0 compatible containers. We are using 2.5 is it?